tests/helpers/test_template.py

```diff
@@ -1072,7 +1066,7 @@ def test_regex_replace(hass):
             """,
         hass,
     )
-    assert tpl.async_render() == "['Home Assistant test']"
+    assert tpl.async_render() == ["Home Assistant test"]
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=6294088)